packages/typespec-test/test/schemaRegistry/generated/typespec-ts/src/models/models.ts

```diff
@@ -41,7 +41,7 @@ export interface PagedVersion {
   /** The Version items on this page */
   value: SchemaVersion[];
   /** The link to the next page of items */
-  nextLink?: string;
+  readonly nextLink?: string;
```

Comment:
Just a little curious what makes the change here?

Reply:
https://github.com/Azure/typespec-azure/pull/231 readonly visibility is added in nextlink.